release/branch.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+
+set -eux
+
+# gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
+
+# # Temporary hack to get around some gcloud credential issues
+# mkdir ~/.docker
+# cp "${DOCKER_CONFIG}/config.json" ~/.docker/
+# export DOCKER_CONFIG=~/.docker
+# gcloud auth configure-docker -q
+
+# PRERELEASE_DOCKER_HUB=${PRERELEASE_DOCKER_HUB:-gcr.io/istio-prerelease-testing}
+# GCS_BUCKET=${GCS_BUCKET:-istio-prerelease/prerelease}
+
+# if [[ -n ${ISTIO_ENVOY_BASE_URL:-} ]]; then
+#   PROXY_OVERRIDE="proxyOverride: ${ISTIO_ENVOY_BASE_URL}"
+# fi
```

Comment:
Remove these commented code?

Reply:
Removed for now. I'm not sure if some of that is needed by the pipeline to make sure the git and gh commands work.